ui/src/routes/devices.$id.settings.access._index.tsx

```diff
@@ -468,3 +468,5 @@ export default function SettingsAccessIndexRoute() {
     </div>
   );
 }
+
+SettingsAccessIndexRoute.loader = loader;
```

Comment:
Fixing the eslint warning about the `export const loader` requires a change here to follow the same pattern as other routes.